### PR TITLE
Cockpit 245

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
https://allisonkarlitskaya.github.io/cockpit-project.github.io/blog/cockpit-245.html

### CODE REVIEW

File: _posts/2021-05-26-cockpit-245.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+slug: cockpit-245
+category: release
+summary: Cockpit with metrics and storage improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 245 and cockpit-podman version 31.
+
+## Release process: No more cockpit-cache tarball
```
Comment (martinpitt) [inline _posts/2021-05-26-cockpit-245.md:15]:
```suggestion
## Release: Drop cockpit-cache tarball
```
I'd suggest moving this to the bottom, as most users won't ever have heard about this, nor care.

File: _posts/2021-05-26-cockpit-245.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+of the keys, so Cockpit now shows both versions to allow key verification for
+both new and old versions.
+
+![Key verification with SHA256 and SHA1 fingerprints](/images/storage-tang-dual-fingerprint.png)
+
+## cockpit-podman: Added Korean translation
+
+A new translation has been added for the Korean language.
+
+Thank you to [simmon](https://translate.fedoraproject.org/user/simmon/) for the efforts.
```
Comment (martinpitt) [inline _posts/2021-05-26-cockpit-245.md:51]:
```suggestion
Thanks to [simmon](https://translate.fedoraproject.org/user/simmon/) for the efforts.
```

**Review by martinpitt — CHANGES_REQUESTED** [review]
Cheers!

File: _posts/2021-05-26-cockpit-245.md (hunk 2 above)
Comment (marusak) [reply _posts/2021-05-26-cockpit-245.md:51]: GH account: https://github.com/simmon-nplob
But weblate account probably makes equal sense here

File: _posts/2021-05-26-cockpit-245.md (hunk 1 above)
Comment (allisonkarlitskaya) [reply _posts/2021-05-26-cockpit-245.md:15]: Good point. I always think about distro packagers when I'm writing these, but it's actually users I should be thinking about...

**Review by garrett — APPROVED** [review]
I have read the content and I approve. :+1:

File: _posts/2021-05-26-cockpit-245.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Thanks to [simmon](https://translate.fedoraproject.org/user/simmon/) for the efforts.
+
+### Try it out
+
+Cockpit 245 and cockpit-podman 31 are available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/245)
+ * [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/FEDORA-2021-)
+ * [Cockpit Fedora 33](https://bodhi.fedoraproject.org/updates/FEDORA-2021-)
```
Comment (martinpitt) [inline _posts/2021-05-26-cockpit-245.md:60]: Still needs these updated.

**Review by martinpitt — CHANGES_REQUESTED** [review]
LGTM, thank you! just a reminder to insert the bodhi URLs -- release is finished.

File: _posts/2021-05-26-cockpit-245.md (hunk 2 above)
Comment (allisonkarlitskaya) [reply _posts/2021-05-26-cockpit-245.md]: I had the GitHub account on the first draft, in fact, but I switched it over to Weblate.

**Pull request merged**